tests/Commands/Wiki/DropDatabaseTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tests\Commands;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+
+class DropDatabaseTest extends TestCase {
+    use DatabaseTransactions;
+
+    public function testWikiNotFound() {
```

Comment:
Looks fine to me, integration testing this is almost always more hassle than it's worth